kaolin/graphics/SoftRenderer.py

```diff
@@ -73,21 +91,22 @@ def __init__(
             viewing_angle: float = 30.,
             viewing_scale: float = 1.0, 
             eye: torch.Tensor = None,
```

Comment:
Actually there is a problem with this argument, it's not used at all.
The definition of self.eye is:
`self.eye = torch.tensor([0, 0, -(1. / math.tan(math.pi * self.viewing_angle / 180) + 1)], device=device)`